spec/sidebar.test.js

```diff
@@ -10,13 +10,13 @@ import { data } from './__mock__/mockProductAndSeller';
 let { product, seller } = data[0];
 
 // jest test
+// Checks if there is any change in the file appearance when it renders

```

Comment:
Hi Stanley, I meant that the comment should become the test explanation. your test explanation says 'testing on the sidebar component' but it doesn't tell me what we are testing. your comment does, but it is not visible to the person who runs the test on the console, in case it fails.